src/components/new-request/search.vue

```diff
@@ -210,41 +225,59 @@
         </template>
 
         <!-- Numbered company bullets -->
-        <v-col v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY" cols="12">
-          <ul class="bullet-points">
-            <li>Your business name will be the Incorporation Number assigned by the Registry.</li>
-            <li>You can change your business name at a later date.</li>
-            <li>It is not possible to request a specific Incorporation Number.</li>
-          </ul>
-        </v-col>
+        <template v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY" cols="12">
+          <v-col v-if="isConversion && !isAlterOnline">
+            <div class="contact-registries">
+              To complete this alteration, please contact us at:
+              <p>
+                <br/>
+                  <v-icon small>mdi-phone</v-icon>  Canada and U.S. Toll Free:
+                  <a href="tel:+1877-370-1033">1-877-370-1033</a>
+              </p>
+              <p><v-icon small>mdi-phone</v-icon>  Victoria Office:
+                <a href="tel:250-370-1033">250-370-1033</a>
+              </p>
+              <p><v-icon small>mdi-email</v-icon>  Email:
+                <a href="mailto:BCRegistries@gov.bc.ca">BCRegistries@gov.bc.ca</a>
+              </p>
+            </div>
+          </v-col>
+          <v-col v-else cols="12">
+            <ul class="bullet-points">
+              <li>Your business name will be the Incorporation Number assigned by the Registry.</li>
+              <li>You can change your business name at a later date.</li>
+              <li>It is not possible to request a specific Incorporation Number.</li>
+            </ul>
+          </v-col>
 
-        <!-- Go to COLIN / Incorporate Now buttons -->
-        <v-col v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY || isFederal" cols="12"
-          class="d-flex justify-center"
-        >
-          <v-btn
-            v-if="showColinButton"
-            class="px-9"
-            id="go-to-colin-button"
-            :href="colinLink"
-            target="_blank"
+          <!-- Go to COLIN / Incorporate Now buttons -->
+          <v-col v-if = "isAlterOnline" cols="12"
+            class="d-flex justify-center"
```

Comment:
Are you sure you need to make this a flex container? I think v-col already is.

Reply:
Have a quick look in this PR whether you need the d-flex class (I think you might not) and update the code or reply accordingly. Thanks!

Reply:
Yes, need this one.